formats/financial_release/frontend/examples/financial_release.json

```diff
@@ -0,0 +1,14 @@
+{
+  "base_path": "/lloyds-share-offer",
+  "content_id": "74314796-eee4-4a66-bc1e-3a2039f680e6",
+  "title": "Financial Release",
+  "format": "financial_release",
+  "locale": "en",
+  "public_updated_at": "2015-12-01T00:00:00.000Z",
+  "details": {
+    "summary": "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,quis nostrud exercitation",
```

Comment:
I think there's no way to make `description` required, but that's not intentional I think. Seems like a shortcoming in the build system. I'll take a look if we can make that possible.

I understand `description` was added to the root instead of the `details` hash because every content item should have one, and they are included in expanded links from the content-store.